### PR TITLE
docs: describe how to run app against real data

### PR DESCRIPTION
**Pull request by geigerzaehler**
We describe how to run the app from source against real user data. This should help users that want to build and run the app from source.

We add this to the development doc and the readme to make it more promentnt. We also move this section to the top of the file to make it more prominent.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -7,6 +7,31 @@
 * [Rustup](https://github.com/rust-lang/rustup)
 * Dependencies of [Cypress](https://docs.cypress.io/guides/getting-started/installing-cypress#System-requirements)
 
+## Running Upstream from source
+
+To start Upstream run `yarn start`.
+
+Running upstream with `yarn start` will use `<repo_root>/sandbox/rad_home` as
+the default `RAD_HOME` value to isolate you development state. To reset the
```
Comment (rudolfs) [inline DEVELOPMENT.md:15]:
```suggestion
the default `RAD_HOME` value to isolate your development state. To reset the
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -7,6 +7,31 @@
...
+application state remove `<repo_root>/sandbox/rad_home` and restart the
```
Comment (rudolfs) [inline DEVELOPMENT.md:16]: Shouldn't we first stop the app, then remove the folder and then start again? 🤔

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -7,6 +7,31 @@
...
+
+If you want to run Upstream against your real user data you need to build the
+application package with `yarn dist`. You can find the packaged application in
+the `dist` folder and run it from there.
+
+### Using the Git remote
+
+Upstream provides the `git-remote-rad` binary to fetch and push Git
+repositories. If you want to use the rad remote in development you need to set
+the environment in your terminal
```
Comment (rudolfs) [inline DEVELOPMENT.md:27]:
```suggestion
the environment in your terminal:
```

File: DEVELOPMENT.md (hunk 2 above)
Comment (geigerzaehler) [reply DEVELOPMENT.md]: https://github.com/radicle-dev/radicle-upstream/pull/2174/commits/ab01f8a2f9fbac3a582fedb10a6e87b019b6155e

**Pull request merged**